packages/host/app/components/operator-mode/stack-item.gts

```diff
@@ -146,28 +152,26 @@ export default class OperatorModeStackItem extends Component<Signature> {
     }
   }
 
-  // TODO replace async action with ember concurrency task
-  @action async copyToClipboard(cardUrl: string) {
-    if (!cardUrl) {
+  copyToClipboard = restartableTask(async () => {
+    if (!this.card.id) {
       return;
     }
     if (config.environment === 'test') {
       return; // navigator.clipboard is not available in test environment
     }
-    await navigator.clipboard.writeText(cardUrl);
-  }
+    await navigator.clipboard.writeText(this.card.id);
+  });
 
-  fetchRealmInfo = trackedFunction(this, async () => {
```

Comment:
not sure if we still need this `trackedFunction`. converted to ember-concurrency

Reply:
actually this is totally cool as a tracked function--a tracked function is a form of a resource. the `this` that is passed into the tracked function binds the lifecycle of the function to the component. I'd suggest moving this back to being a tracked function